### DISCUSSION TITLE
GR-LotResult-Section in notice subtype 28

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#874] [Q&A] [answered]
GR-LotResult-Section does not exist in notice subtype 28. Is this intentional? All other direct award pre-announcements (subtype 25-27) contain this section.

**Comment by YvesJo** [comment, marked as answer]
Hi,
This is compliant with the Regulation Annex:
![image](https://github.com/OP-TED/eForms-SDK/assets/42802091/46823aa9-c013-4c1e-8223-f5848a79c13d)

If you believe there is an issue there, please get in touch with DG GROW directly

**Reply by JanEmde** [reply]
Ok. Do you have a direct contact at DG Grow?

**Reply by YvesJo** [reply]
You could document your concern here: https://code.europa.eu/groups/eproc/-/issues

**Reply by JanEmde** [reply]
Thanks a lot!

**Reply by JanEmde** [reply]
@YvesJo FYI: In fact, the annex is incorrect here. This should be corrected in the future.
https://code.europa.eu/eproc/eforms/crs/-/issues/38

**Reply by YvesJo** [reply]
Thanks for sharing.
This is quite surprising as there may be no Framework Agreement for Directive 23, hence no other information in the Business Group (BG-137) besides BT-13713.
Also it is still possible to identify the winner(s) of a Lot in absence of LotResult, in the Business Group Contract (BG-310), there is a direct link to the Tender(s) that lead to the contract (BT-3202) and the Tender(s) (BG-320) also specify the Lot they have been submitted for (BT-13714)
The case is similar for Contract Modifications.

**Reply by JanEmde** [reply]
Besides BT-13713 there are a few OPTs in the BG, right? At least OPT-301-LotResult-Financing and OPT-301-LotResult-Paying.

**Reply by YvesJo** [reply]
Indeed, possibility to specify roles does not appear in the annex and could be a reason to allow for the Lot Result

**Reply by JanEmde** [reply]
Another problem is that "_the winning Tender TEN-0001 should be referenced from both the LotResult (OPT-320-LotResult : "Tender Identifier Reference") and from the SettledContract (BT-3202-Contract : "Contract Tender ID (Reference)"). Without these Fields present, the section "6.1.2 Information about winners" under "6.1 Result Lot Identifier: LOT-0001" will be empty._" (This is a comment from a discussion on the View of Direct award preannouncements: https://github.com/OP-TED/eForms-SDK/issues/872). This must be changed if subtype 28 does not cotain the section.

**Reply by YvesJo** [reply]
This is a bug fixed with SDK 1.11 that will also be backported, so it shouldn't be seen as a justification for having LotResult as this information may be built without LotResult. For forms 25 to 27, Lot Result is also only optional and may therefore not be present.

**Reply by JanEmde** [reply]
In my understanding the bugfix refers only to the display of the information on the direct award (BT-136-Procedure and BT-135-Procedure). If this is meant differently, I do not understand the hint about the missing referencing.

**Reply by YvesJo** [reply]
The hints provided in the other discussion had to be considered in the context of the example provided with the question where the LotResult was present in the XML while some information was missing in it (which couldn't have been the case for form 28).
In absence of LotResut, this is the connections SettledContract => LotTender => Lot (or Group of Lots => Lots) and LotTender => Tendering Party => Organisation that help build the LotResult on the displayed notice.

**Reply by JanEmde** [reply]
Ah got it. Thank you!
